src/components/contact-form/contact-form.jsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import RightArrow from 'svg/right-arrow/right-arrow';
+import styles from './contact-form.module.scss';
+import EmailInput from './email-input';
+import PaintingsInput from './paintings-input';
+import InputShell from './input-shell';
+
+export default function ContactForm() {
+  const [isToggled, setIsToggled] = useState(false);
+  const {
+    register,
+    handleSubmit,
+    setValue,
+    formState: { errors },
+  } = useForm();
+
+  const formInputs = [
```

Comment:
I understand what you're doing here, but do you think it's necessary? Do you find this more readable than laying out all the JSX without a map?